src/redux/reducers/articleReducer.js

```diff
@@ -0,0 +1,10 @@
+import { GET_ARTICLES } from '../actions/actionTypes';
+import initialState from './initialState';
```

Comment:
@leksyib why the Hound errors?

Reply:
@9jaswag  Hound isn't seeing the houndci config files since Samuel's PR has not been merged. But I ran the lint command before pushing so I'm sure there's no hound violation.